pkg/types/installconfig.go

```diff
@@ -373,6 +379,24 @@ type BootstrapInPlace struct {
 	InstallationDisk string `json:"installationDisk"`
 }
 
+// Capabilities selects the managed set of optional, core cluster components.
+type Capabilities struct {
+	// baselineCapabilitySet selects an initial set of
+	// optional capabilities to enable, which can be extended via
+	// additionalEnabledCapabilities.  If unset, the cluster will
+	// choose a default, and the default may change over time.
```

Comment:
Since the installer is installing a particular static version, should we removed the language about the default changing?

Reply:
+1